<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="whatsThis">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>网页模板导入</string>
       </attribute>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>851</width>
          <height>81</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="source_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>选择文件夹</string>
         </property>
         <property name="id" stdset="0">
          <number>10001</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>源文件路径:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="source_dir">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>10</y>
           <width>600</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>851</width>
          <height>81</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>选择文件夹</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>template路径:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="template_dir">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>600</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>网页文件夹命名:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="web_name">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>300</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>851</width>
          <height>81</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>选择文件夹</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>static路径:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="static_dir">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>600</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>1</string>
    </property>
   </widget>
   <addaction name="menu1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>source_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>main_btn_click()</slot>
  <slot>Tab1_TemplateDir(QString)</slot>
  <slot>select_dir()</slot>
 </slots>
</ui>
